server/todo-api/schemas/to-do-schema.ts

```diff
@@ -0,0 +1,15 @@
+import { model, Schema } from 'mongoose';
+import { ToDo } from '../model/todo-model';
+
+const ToDoSchema = new Schema<ToDo>({
+  createdBy: { type: Schema.ObjectId },
+  isCompleted: { type: Boolean, default: false },
+  description: { type: String },
+  priority: { type: Number, max: 10, min: 0 },
+  title: { type: String, required: true },
+  dueDate: { type: Date },
+});
+
+const ToDoModel = model('todo', ToDoSchema);
+
+export default ToDoModel;
```

Comment:
Es mejor usar named exports en el caso de los modelos para que en otras partes del codigo se sepa de que se esta hablando. Ademas se diferencia de la interfaz, no es conveniente exportar una interfaz y objeto con el mismo nombre.
```suggestion
export const ToDoDBModel = model('todo', ToDoSchema);
```